src/app/components/Curation/index.tsx

```diff
@@ -66,25 +65,30 @@ export default ({
 
   const isFirstCuration = position === 0;
   const curationSubheading = title || topStoriesTitle;
-  const id = idSanitiser(curationSubheading);
+  const id =
```

Comment:
**Note for reviewers:** The ID will always be unique, because `nthCurationByStyleAndProminence` gives a unique number to each component with the same visual style & prominence